spec/Expressions.tex

```diff
@@ -298,6 +295,14 @@ \section{Precedence and Associativity}
 \hline
 \verb@new@ & right & initializer call \\
 \hline
+\verb@owned@ & right & apply management strategy to a class \\
+\verb@shared@ & right & \\
+\verb@borrowed@ & right & \\
+\verb@unmanaged@ & right & \\
+\hline
+\verb@?@ & left & compute a nilable class type \\
+postfix \verb@!@ & left & assert non-nilable and borrow \\
```

Comment:
prepend `postfix` to `?` as well, for consistency
remove `left` in the second line above